src/modifiers/position.py

```diff
@@ -14,8 +14,9 @@
     "before": {"position": "before"},
     "start of": {"position": "before", "insideOutsideType": "inside"},
     "end of": {"position": "after", "insideOutsideType": "inside"},
-    "above": {"position": "before", **LINE.json_repr},
-    "below": {"position": "after", **LINE.json_repr}
+    # Disabled for now
```

Comment:
```suggestion
    # Disabled for now because "below" can misrecognize with "blue" and we may move away from allowing positional modifiers in arbitrary places anyway
```